interface/patient_file/summary/demographics.php

```diff
@@ -1306,7 +1306,7 @@ function setMyPatient() {
     // Show Clinical Reminders for any user that has rules that are permitted.
     $clin_rem_check = resolve_rules_sql('','0',TRUE,'',$_SESSION['authUser']);
     if (!empty($clin_rem_check) && $GLOBALS['enable_cdr'] && $GLOBALS['enable_cdr_crw'] &&
-        acl_check('patients', 'reminder'))
```

Comment:
I think it mistake.. it should be 'alert' ACO for clinical reminders widget.
![image](https://cloud.githubusercontent.com/assets/17809866/26091794/125f3886-3a16-11e7-8385-0e089555f13a.png)